.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Documentation
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+    name: Build HTML documentation
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install documentation dependencies
+        run: |
+          python -mpip install -r doc/requirements.txt
```

Comment:
I understand that we are keeping dependences to create documentation on one place and dependences for the package _itself_ on the `toml` file. This is so because you only need  this dependances at push / PR time correct?